tests/python/pants_test/core_tasks/test_list_goals.py

```diff
@@ -99,8 +99,9 @@ def execute(self):
     with self.assertRaises(GoalError) as ctx:
       goal.install(TaskRegistrar(task_name, OtherNoopTask))
 
-    self.assertIn('foo', ctx.exception.message)
-    self.assertIn(self._LIST_GOALS_NAME, ctx.exception.message)
+    exception_message = ' '.join(ctx.exception.args)
```

Comment:
Py3 doesn't have `exception.message`. Both have `exception.args` though, which is a tuple we convert to a string.

Reply:
I believe `str(ctx.exception)` consistently gives `exception.message` on both Pythons

Reply:
You're right! Good catch.